docs/reference/commandline/service_rollback.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "service rollback"
+description: "The service rollback command description and usage"
+keywords: "service, rollback"
+---
+
+<!-- This file is maintained within the docker/cli Github
+     repository at https://github.com/docker/cli/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+# service rollback
+
+```Markdown
+Usage:	docker service rollback SERVICE
+
+Rollback a service
+
+Options:
+  -d, --detach       Exit immediately instead of waiting for the service to converge (default true)
+      --help         Print usage
+  -q, --quiet        Suppress progress output
+```
+
+## Description
+
+Rollback a specified service from the swarm. This command has to be run
+targeting a manager node.
+
+## Examples
+
+Rollback the `redis` service:
+
```

Comment:
Agreed, this could use some additional information (also strip down the output to show just the relevant information, as mentioned in my other comments), something like;

    ### Roll back a service to a previous version

    In this example, a service is created using the `redis:3.0.6` image,
    updated to use the `redis:3.0.7` image, and then rolled back to its
    previous version.

    Create a service using the `redis:3.0.6` image:

    ```bash
    $ docker service create --name my-service redis:3.0.6
    ```

    Confirm that the service is running:

    ```bash
    $ docker service ps my-service

    ID                  NAME                IMAGE               NODE                DESIRED STATE       CURRENT STATE           ERROR               PORTS
    s2xak85rftfc        my-service.1        redis:3.0.6         moby                Running             Running 4 seconds ago
    ```

    Update the service to use the `redis:3.0.7` image:

    ```bash
    $ docker service update --image=redis:3.0.7 my-service
    ```

    The service is now updated to use the `redis:3.0.7` image:

    ```bash
    $ docker service ps my-service

    ID                  NAME                IMAGE               NODE                DESIRED STATE       CURRENT STATE                    ERROR               PORTS
    7jtl9bxnxnx1        my-service.1        redis:3.0.7         moby                Running             Running less than a second ago
    s2xak85rftfc         \_ my-service.1    redis:3.0.6         moby                Shutdown            Shutdown 10 seconds ago
    ```

    Roll back the service to its previous version. Rolling back the service
    reverts all changes made to the service in the last update. In this
    example, rolling back the service changes the service's image back to
    `redis:3.0.6`.

    ```bash
    $ docker service rollback my-service

    $ docker service ps my-service

    ID                  NAME                IMAGE               NODE                DESIRED STATE       CURRENT STATE            ERROR               PORTS
    8gwl1xoi0xh3        my-service.1        redis:3.0.6         moby                Running             Running 3 seconds ago
    7jtl9bxnxnx1         \_ my-service.1    redis:3.0.7         moby                Shutdown            Shutdown 4 seconds ago
    s2xak85rftfc         \_ my-service.1    redis:3.0.6         moby                Shutdown            Shutdown 2 minutes ago
    ```